### PR TITLE
Add periodic job for kubevirt-velero-plugin

### PR DESCRIPTION
**Pull request by brybacki**
Signed-off-by: Bartosz Rybacki <brybacki@redhat.com>

### CODE REVIEW

**Comment by brybacki** [comment]
error says no dashboard - this requires another change:
https://github.com/kubevirt/project-infra/pull/1960

**Comment by brybacki** [comment]
rebased

**Comment by brybacki** [comment]
@awels Can you take a look?

**Comment by brybacki** [comment]
yes, it requires kubevirtci, it starts some fedora with guest agent vms, does backup restore and so on.

**Comment by awels** [comment]
looks like I can't approve, @phoracek could you approve?

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-velero-plugin/kubevirt-velero-plugin-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-gcs-credentials: "true"
+    preset-shared-images: "true"
+    preset-kubevirtci-quay-credential: "true"
```
Comment (phoracek) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-velero-plugin/kubevirt-velero-plugin-periodics.yaml:16]: ditto

Comment (brybacki) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-velero-plugin/kubevirt-velero-plugin-periodics.yaml:16]: the jon only uses kubevirtci quay as a read only, so I suppose not needed

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-velero-plugin/kubevirt-velero-plugin-periodics.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  annotations:
+    testgrid-dashboards: kubevirt-velero-plugin-periodics
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-gcs-credentials: "true"
```
[the remark this answers is not in the document]
Comment (brybacki) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-velero-plugin/kubevirt-velero-plugin-periodics.yaml:14]: not this one

[the remark this answers is not in the document]
Comment (brybacki) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-velero-plugin/kubevirt-velero-plugin-periodics.yaml]: removed

**Comment by phoracek** [comment]
/lgtm
/approve

Thanks

**Pull request merged**